src/workflows/steps.js

```diff
@@ -48,10 +48,30 @@ export function checkHasDevelopBranch( state ) {
 	} );
 }
 
-export function askPrereleaseIdentifier( state ) {
+export function setPrereleaseIdentifier( state ) {
 	const { identifier } = state;
+	const redundantIdentifierPrefixes = [
+		"defect",
+		"feature",
+		"rework"
+	];
+
+	const cleanIdentifier = target => {
```

Comment:
Yes you were right..._MUCH_ simpler. Thanks for pointing that out. This update has been pushed.